plugins/kafka/src/components/ConsumerGroupOffsets/ConsumerGroupOffsets.tsx

```diff
@@ -57,6 +57,17 @@ const generatedColumns: TableColumn[] = [
       return <>{row.groupOffset ?? ''}</>;
     },
   },
+  {
+    title: 'Lag',
+    field: 'lag',
+    render: (row: Partial<TopicPartitionInfo>) => {
+      let lag = undefined;
+      if (row.topicOffset && row.groupOffset) {
+        lag = +row.topicOffset - +row.groupOffset;
+      }
+      return <>{lag ?? ''}</>;
+    },
+  },
```

Comment:
Could we add test cases for this? :)

Reply:
Hi @OrkoHunter I added a test case. Please take a look and let me know what you think :)